cyhy/core/geoloc.py

```diff
@@ -39,3 +40,12 @@ def lookup(self, ip):
             return (response.location.longitude, response.location.latitude)
         except AddressNotFoundError:
             return (None, None)
+
+    def check_restricted_ip(self, ip):
+        try:
+            response = self.__reader.city(str(ip))
+            if response.country.name in RESTRICTED_COUNTRIES:
+                return True,response.country.name
+            return False,None
```

Comment:
The `False` return has been moved to the end of the method.

```suggestion
                return True, response.country.name
```